.travis/check_relevant_lint.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
This one gets overwritten to `bash` by `dev-kit` on the next run.

Why did you change this?